deel/influenciae/rps/rps_l2.py

```diff
@@ -59,158 +59,17 @@ def __init__(
             lambda_regularization: float,
             scaling_factor: float = 0.1,
             epochs: int = 100,
-            layer_index: int = -2,
+            layer_index: int = -1,
```

Comment:
Considering the theorem, isn't it better that this value is always -1? (*i.e.* not an init param)

Reply:
Because of the way I'm splitting the model into two graphs in the new abstraction's constructor, it gives the same results on the tests that I've run.